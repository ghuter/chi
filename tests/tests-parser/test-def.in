A :: define X [i32];
B :: define X [i32, String];
B :: define X [i32, String] (M: HashI32);
