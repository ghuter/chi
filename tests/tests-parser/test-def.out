SMODDEF(A, [i32], (), | |) : X
SMODDEF(B, [i32, String], (), | |) : X
SMODDEF(B, [i32, String], (M: HashI32[]), | |) : X
